titanic.ipynb

```diff
@@ -43,9 +43,6 @@
         "# Load the Titanic dataset\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 17_
```python

# Create a countplot of the man/woman/child classification
sns.countplot(data=titanic, x='who', hue='alive', pallete='bright')

# Add some labels and title
plt.title('Survival Rates by Man/Woman/Child')
plt.xlabel('Person Type')
plt.ylabel('Count')
```
plt.ylabel

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/26)._